knimin/lib/tests/test_geocoder.py

```diff
@@ -76,12 +76,12 @@ def test_geocode_nonmock(self):
         obs = geocode('9500 Gilman Dr, La Jolla, CA')
         exp = Location('9500 Gilman Dr, La Jolla, CA', 32.8794239,
                        -117.2369135, 114.7895736694336, 'San Diego', 'CA',
-                       '92093', 'United States')
+                       '92037', 'United States')
```

Comment:
For future reference, its good practice to add in your commit message why these values changed.